styles/styles.css

```diff
@@ -248,6 +248,10 @@
     main {
       padding-top: 8rem;
     }
+
+    :root {
+      --is-large-screen: 1;
+    }
```

Comment:
I think this should be moved to `simplified-styles.css` as after block simplification, the intent is to remove this.

Reply:
Ok